markdown/events/2023/hackathon-march-2023/uk-google.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Hackathon - March 2023 (Google Academy)
+subtitle: Local node of the nf-core hackathon at Google Academy, London.
+type: hackathon
+start_date: '2023-03-27'
+start_time: '10:00 CEST'
+end_date: '2023-03-29'
+end_time: '18:00 CEST'
+location_name: Google Academy, London
+# address:
+# location_url:
+# location_latlng: [xxx, xxx]
```

Comment:
```suggestion
address: 123 Buckingham Palace Rd, London SW1W 9SH
location_url: https://www.wired.co.uk/article/google-digital-skills-academy
location_latlng: [51.493721, -0.146830]
```

Reply:
Thanks @christopher-hakkaart! Can you just check the `location_lating` entry looks good? Otherwise good to merge.